docs/developer-tools/ThrowHelper.md

```diff
@@ -0,0 +1,150 @@
+---
+title: ThrowHelper
+author: Sergio0694
+description: Helper methods to efficiently throw exceptions
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, debug, net core, net standard
+dev_langs:
+  - csharp
+---
+
+# ThrowHelper
+
+The [ThrowHelper](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.diagnostics.ThrowHelper) class is a helper type that can be used to efficiently throw exceptions. It is meant to support the [Guard](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.diagnostics.guard) APIs, and it should primarily be used in situations where developers need fine-grained controls over the exception types being thrown, or over the exact exception message to include.
+
+## How it works
```

Comment:
Should probably stick to standard `Syntax` title here for consistency?

Reply:
Done in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/370/commits/dc598defa183fbc19dca80e5a9d24f17bffafa0b 👍